### ISSUE TITLE
Selected directory is already assigned to mergin project

### ISSUE THREAD

**Issue by kevin-garcia** [lutraconsulting/qgis-mergin-plugin#78]
I created a mergin project from a local directory, but needed to make changes to a form before using it. So, I deleted the project from mergin, changed the form in QGIS, saved the project locally, and tried to create a new project using the same local directory I used for the initial (now deleted) mergin project. 

I get this error, even though the mergin project I had created from this directory was deleted:

![image](https://user-images.githubusercontent.com/16250189/69971094-4c7e5400-1517-11ea-8c04-c8f7a6d23ebf.png)

**Comment by saberraz** [comment]
There is a hidden folder called `.mergin`. If you set your file explorer to show hidden files and folders, you can then delete this folder to disassociate it to any project.

**Comment by kevin-garcia** [comment]
I noticed that right after my initial post, when I went and looked at the local directory, but thanks for the reply! I was able to delete it and re-create the project successfully. This is a simple enough work around that I'll mark this "closed".

It's probably a low-priority thing, but maybe in the future the pop-up could ask if you want to replace the existing project with a new one, and if "yes" then it would replace the .mergin folder for you.

(This is an amazing project, by the way. I'd been looking for an open-source field collector to integrate with QGIS, and I am really excited to have discovered mergin/input. Thank you so much for creating this.)
